python/res/enkf/ert_workflow_list.py

```diff
@@ -98,7 +103,8 @@ def getContext(self):
         return self._get_context()
 
     def __str__(self):
-        return 'ErtWorkflowList with jobs: %s' + str(self.getJobNames())
+        return 'ErtWorkflowList'
+        # return 'ErtWorkflowList with jobs: %s' + str(self.getJobNames())
```

Comment:
Why not return the job names?

Reply:
That was due to debugger - now putting them back